frontend/src/components/Order/components/ChangeDataBtn/ChangeDataBtn.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '../../../ui-kit/components/Button';
+import { ICONS } from '../../../ui-kit/icons';
+import styles from './ChangeDataBtn.scss';
+
+const ChangeDataBtn = ({ sectionState, handleChangeData }) => {
+  const initialState = sectionState;
+
+  return (
+    <>
+      {initialState && (
+        <div className={styles.buttonChange}>
+          <Button
+            variant="no-border"
+            label="Змінити"
+            icon={<ICONS.PencilIcon />}
+            onClick={() => handleChangeData()}
```

Comment:
It's better to keep it simple. OnClick={handleChangeData}

Reply:
I don't use component ChangeDataBtn. I deleted it.